istio-control/istio-autoinject/templates/sidecar-injector-configmap.yaml

```diff
@@ -9,304 +9,12 @@ metadata:
     app: sidecar-injector
     istio: sidecar-injector
 data:
-  mesh: |-
-    # No longer used
+  values: |-
```

Comment:
@howardjohn does this values field make sense now that values.yaml is spread all about the repo?  We will only record the top level values.yaml as the code exists today.

Reply:
AFAIK this records the merged values used to install the sidecar ( which may be different from other components - we don't share anything )

Reply:
As long as everything we access is in the sidecars values or the global values this will work